### PR TITLE
[Docs] advanced offloading

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This pr adds documentation on advanced offloading

### CODE REVIEW

File: docs/advanced/peering/advanced-offloading.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
+# Offloading in depth
+
+## Overview
+
```
Comment (claudiolor) [inline docs/advanced/peering/advanced-offloading.md]: I would add a note telling that this is an advanced proceed and to stick with `liqoctl offload` when there aren't specific needs. (Like, for example, we have in [this page](https://github.com/liqotech/liqo/blob/docs/docs/advanced/peering/inter-cluster-authentication.md?plain=1#L18))

File: docs/advanced/peering/advanced-offloading.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+This document will go over the process of acquiring resources and making them available as a `Node` in the consumer cluster.
+
+You can add a `VirtualNode` to a consumer cluster in two different ways:
+
+1. By creating a `ResourceSlice` in the tenant namespace in the consumer cluster.
+2. By creating a `VirtualNode` in the consumer cluster.
+
+Note that the `ResourceSlice` method is the **preferred way** to add a `VirtualNode` to a consumer cluster, but it requires the *Authentication module* to be enabled (that is enabled by default).
```
Comment (claudiolor) [inline docs/advanced/peering/advanced-offloading.md]: I would add in which case the authentication module is enabled by default (e.g. when peering is creating with the `liqoctl peer` command)

File: docs/advanced/peering/advanced-offloading.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+This command will create a `ResourceSlice` named `mypool` in the consumer cluster, and it will be associated with the `cool-firefly` cluster.
+
+If no resource is specified, the provider cluster will fill them with the default values.
+You can specify the resources you want to acquire by adding:
+
+* `--cpu` to specify the amount of CPU.
+* `--memory` to specify the amount of memory.
+* `--pods` to specify the number of pods.
+
+For other custom resources, you can use the `-o yaml` flag for the `liqoctl create resourceslice` command and edit the `ResourceSlice` spec manifest.
```
Comment (claudiolor) [inline docs/advanced/peering/advanced-offloading.md]:
```suggestion
For other custom resources, you can use the `-o yaml` flag for the `liqoctl create resourceslice` command and edit the `ResourceSlice` spec manifest before applying it.
```

File: docs/advanced/peering/offloading-in-depth.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+This command will create a `ResourceSlice` named `mypool` in the consumer cluster, and it will be associated with the `cool-firefly` cluster.
+
+If no resource is specified, the provider cluster will fill them with the default values.
+You can specify the resources you want to acquire by adding:
+
+* `--cpu` to specify the amount of CPU.
+* `--memory` to specify the amount of memory.
+* `--pods` to specify the number of pods.
+
+For other custom resources, you can use the `-o yaml` flag for the `liqoctl create resourceslice` command and edit the `ResourceSlice` spec manifest.
```
Comment (claudiolor) [inline docs/advanced/peering/offloading-in-depth.md:59]:
```suggestion
To add other resources like `ephemeral-storage`, `gpu` or any other custom resources, you can use the `-o yaml` flag for the `liqoctl create resourceslice` command and edit the `ResourceSlice` spec manifest before applying it.
```

File: docs/advanced/peering/offloading-in-depth.md [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+
+```bash
```
Comment (claudiolor) [inline docs/advanced/peering/offloading-in-depth.md:61]: I would add a note in the snippet telling where the commands should be executed. Like we do [here](https://github.com/liqotech/liqo/blob/docs/docs/advanced/peering/inter-cluster-authentication.md?plain=1#L64)

File: docs/advanced/peering/offloading-in-depth.md [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+
+You can revert the process by deleting the `VirtualNode` in the consumer cluster.
+
+```bash
+kubectl delete virtualnode mynode -n liqo-tenant-cool-firefly
+```
+
+## Multiple VirtualNodes
+
+You can create multiple `VirtualNodes` associated with the same provider cluster.
```
Comment (claudiolor) [inline docs/advanced/peering/offloading-in-depth.md:238]: In each of the section I would add a brief example explain when that case might be useful. For example, in this case, why should I create multiple virtual node?

**Review by fra98 — APPROVED** [review]

**Pull request merged**